tests/BlazingSQLTest/EndToEndTests/coalesceTest.py

```diff
@@ -335,18 +337,22 @@ def executionTest(queryType):
 
             select * from lastjoin
             """
-            runTest.run_query(
-                bc,
-                drill,
-                query,
-                queryId,
-                queryType,
-                worder,
-                "",
-                acceptable_difference,
-                use_percentage,
-                fileSchemaType,
-            )
+
+            # TODO: Too expensive with nulls reason: timeout
+            testsWithNulls = Settings.data["RunSettings"]["testsWithNulls"]
+            if testsWithNulls != "true":
```

Comment:
I wonder if this is timing out for the same reason TEST_13 is failing. How is TEST_13 failing? Are there a lot more rows on the BSQL result? is it about 1.5x - 2x more rows? or more like 50x?

Reply:
The error changed for this test, see https://github.com/BlazingDB/blazingsql/issues/1405